src/java/org/apache/cassandra/service/accord/txn/TxnQuery.java

```diff
@@ -154,10 +152,6 @@ public Result compute(TxnId txnId, Timestamp executeAt, Seekables<?, ?> keys, @N
         Epoch epoch = Epoch.create(executeAt.epoch());
         if (transactionIsInMigratingOrMigratedRange(epoch, keys))
         {
-            // Fail fast because we can't be sure where this request should really run or what was intended
```

Comment:
I'd lean towards leaving it out, and I'm not sure what it would look like if we left it. The original check seemed to be more an artifact of LWTs being converted and interop/migration being implemented independently of each other. With the schema config, there isn't really an LWT strategy, your table is either using accord or not, and that informs how LWTs are handled.

Reply:
The check if whether the key is in a migrating or migrated range from Accord to Paxos, so if the strategy for the table is `TransactionalMode.full` then the key shouldn't be in a migrating or migrated range *from* Accord and still be attempting to execute on Accord.

This is also another place where we are using the `ConsensusMigrationState` from the current cluster metadata and not the epoch the query executes at! I have a patch set that fixes this in another place I need to update.

Reply:
> The check if whether the key is in a migrating or migrated range from Accord to Paxos, so if the strategy for the table is TransactionalMode.full then the key shouldn't be in a migrating or migrated range from Accord and still be attempting to execute on Accord.

If we do migrations between interop modes then this check would be overly defensive